ios/PurchasesHybridCommon/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.11.0</string>
+	<string>1.11.1</string>
```

Comment:
should this be a minor since we added a property?

Reply:
I thought about it, but it's not a _new_ property - the property was added in `1.10.0`, it's just that for Android the value was always `"unknown"`